### ISSUE TITLE
getContactsMatchingString not fetching urlAddresses 

### ISSUE THREAD

**Issue by archilo20** [rt2zz/react-native-contacts#479]
Hi I’m trying to get a contact based on the string, the method works correctly and returns the contact, however the urlAddresses array always comes empty even if the phone contact has urls.
Please help.
RN: 0.59.5
RNContacts: 5.0.4
```es
{
company: ""
department: ""
displayName: "Jhon Fox"
emailAddresses: (2) [{…}, {…}]
familyName: ""
givenName: "Jhon Fox"
hasThumbnail: false
jobTitle: ""
middleName: null
note: ""
phoneNumbers: (3) [{…}, {…}, {…}]
postalAddresses: (2) [{…}, {…}]
prefix: null
rawContactId: "2140"
recordID: "2141"
suffix: null
thumbnailPath: ""
**urlAddresses: []**
__proto__: Object
}
```

urlAddresses array comes empty.

Thanks

**Comment by morenoh149** [comment]
a url fix landed in 5.0.5 https://github.com/rt2zz/react-native-contacts/commit/10474b9d37d094988df4b29bf36cd559fa95e856 try using the latest

**Comment by archilo20** [comment]
Thank you!
